### PR TITLE
Adds a new transfer center variant

### PR DESCRIPTION
**Pull request by maxion12345**
### _**THE POOL! THE POOL! THE POOL!**_

# Document the changes in your pull request

Adds a new transfer center with a new method: drowning! 

Features a pool with a wonderful selection of chemicals to dump into so your prisoner can have all the fun he wants splashing around. You can even beat him with a soggy pool noodle prior to dunking him into **_THE POOL_**

68747470733a2f2f6d64622e616666656374656461726330372e636f2e756b2f46696c65732f3132303936363333362f353233373933333639312f302f61667465722e706e67

# Wiki Documentation

n/a

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
rscadd: Added THE POOL to the transfer center rotation.
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
I don't know if this is "wrong" but your templates seem to be inconsistent with what we currently have.
Room variations don't include the walls themselves, only the stuff inside.

Unless someone made the execution room different when they created the other templates.

**Comment by nmajask** [comment]
I was working on the same thing last night, and I also put a pool noodle on the table.

> I don't know if this is "wrong" but your templates seem to be inconsistent with what we currently have. Room variations don't include the walls themselves, only the stuff inside.
> 
> Unless someone made the execution room different when they created the other templates.

Transfer center does, because some change the walls like the one with the blast doors.

**Comment by maxion12345** [comment]
> 

i copypasted from the other one, the mapbot doesn't show the APCs and whatnot because they are pixel-shifted over the level border

**Comment by Hopekz** [comment]
> > 
> 
> i copypasted from the other one, the mapbot doesn't show the APCs and whatnot because they are pixel-shifted over the level border

Wack

**Comment by maxion12345** [comment]
heres the full
uFTrgX3
and it fits here
fR6476E

**Review by patpol4 — APPROVED** [review]
The bed looks stupid in the water but it should work, awaiting approval for the code changes.

File: _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+	pixel_x = 2;
+	pixel_y = 8
+	},
+/obj/item/twohanded/required/pool/pool_noodle{
+	pixel_x = 8;
+	pixel_y = -7
+	},
+/obj/item/reagent_containers/glass/beaker{
+	list_reagents = list(/datum/reagent/consumable/condensedcapsaicin = 30);
+	name = "beaker of capsician";
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm:207]: Capsician

Comment (patpol4) [reply _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm:207]: Good catch

File: code/modules/pool/pool.dm [hunk 2]
```diff
@@ -182,6 +182,10 @@ Place a pool filter somewhere in the pool if you want people to be able to modif
 	icon = 'icons/obj/pool.dmi'
 	icon_state = "ladder"
 	pixel_y = 12
+	var/reversed = null
```
Comment (Bibby0110) [inline code/modules/pool/pool.dm]:
```suggestion
	var/reversed = FALSE
```

**Review by Bibby0110 — APPROVED** [review]

File: _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm (hunk 1 above)
Comment (maxion12345) [reply _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm:207]: Fuck

Comment (Hopekz) [reply _maps/RandomRuins/StationRuins/BoxStation/transfer10.dmm:207]: image

**Pull request merged**